Add a wiki page for GeoJson - and rename User/GlobeBrowsing to User/Globes (#29)
OpenSpace/openspace.github.io: docs/users/globebrowsing/geojson-layers.md

Some examples of features are shown below (polygons, points with textures, and lines).

![geojson examples](./images/geojson/geojson_examples.jpg)

Each GeoJson file loads a *collection* of features, which will make up a group of rendered features in OpenSpace, that can be toggled on/off, et cetera. In OpenSpace, the contents of a file are loaded into one property owner that will show up under the globe in the user interface, under "GeoJson". Due to the similarity to [Globebrowsing Layers](./working-with-layers.md), we refer to these as "GeoJson Layers", or "Globe Geometry Layers".

## Adding GeoJson features to a globe
There are two ways of adding GeoJson features to a selected globe: either directly from a .geojson file or by specifying a table in an OpenSpace asset. Once added, all content in a GeoJson file will appear as a component under "GeoJson" in the scene menu, as part of the globe it was added to.

### Creating a GeoJson asset

> The preferred way to load a GeoJson file is to create an asset. This lets you load the file and add it to the correct globe, already at startup. It also lets you easily customize the visual properties of the rendered feature.

Reviewer: Avoid saying it is easy to do things, it might not be according to the reader.

| Extrude             | `extrude`                        | boolean (`true`/`false`)                 | If true, extrude the geometry to intersect the globe. Lines/polygons will be extruded with polygons, and points with lines |
| AltitudeMode        | `altitudeMode`                   | string (`"absolute"` or `"relativeToGround"`) | Decides how any height values of the geo coordinates should be interpreted. "Absolute" means that the height is interpreted as the height above the reference ellipsoid (sea level), while "RelativeToGround" takes the height map into account |
| **Points Specific**          |                   |                              |  |
| PointSize           | `point-size`                     | float (> 0)                              | A value that will be used to decide the size of rendered points. The size will be scaled based on the bounding sphere of the globe   |
| PointTexture        | `texture`, `sprite`, `point-texture` | string (path to a texture)           | A texture to be used for rendering points |

> | PointTextureAnchor  | `point-anchor`, `anchor`         | string (`"bottom"` or `"center"`)        | Decides the placement of the point texture in relation to the position. Default is a the bottom of the texture, but it can also be put at the center |

Reviewer: ```suggestion
| PointTextureAnchor  | `point-anchor`, `anchor`         | string (`"bottom"` or `"center"`)        | Decides the placement of the point texture in relation to the position. Default is at the bottom of the texture, but it can also be put at the center |
```

In addition to the visual properties in the GeoJson files, some regular OpenSpace properties can be set for any loaded features. These can be used to navigate to a feature, change the visuals to match the projection system you are currently using, or just alter the look in general. These can not be set from a .geojson file, but through the Scene menu in the user interface, or in an asset.

#### Height offset, opacity, and scale
A few helper properties have been added to simplify changing some aspects of the visuals of the entire collection at once:

- *Opacity* - change the opacity of all features in a collection
- *Height Offset* - move the geometry up/down in relation to the reference surface (based on the height map or sea level, depending on chosen Altitude Mode)
- *Point Size Scale* - increase/decrease the size of any points in the collection
- *Line Width Scale* - increase/decrease the width of any rendered lines in the collection

#### Alignment of points
For points, it is possible to change how the texture planes align against the globe surface. There are four different options:

| Option           | Value | Description |
| ---------------- | :---: | ----------- |

> | Camera Direction | 0     | Rotate the plane to face the camera view direction and alight the up-driection with the camera's up. Suitable for planar displays |

Reviewer: ```suggestion
| Camera Direction | 0     | Rotate the plane to face the camera view direction and aline the up-direction with the camera's up. Suitable for planar displays |
```

Author: *align

| Camera Position  | 1     | Use the camera's up direction, but rotate the plane to face the camera position instead of view direction. Suitable for spherical displays, such as domes or planetariums |
| Globe Normal     | 2     | Use the out-driection of the globe as the up direction |
| Globe Surface    | 3     | Align the plane flat along the globe surface |

Here are examples of points aligned with the Camera Direction, Globe Normal and Globe Surface options, respectively. The points have been extruded with a line that intersects the globe surface
![point alignment examples](./images/geojson/points_transparent.png)

#### Light sources

> Polygon geometry can be lit using an OpenSpace light source (for example the Sun, or the Camera). Per default, a camera-based light source is added for all features.

Reviewer: How would it look like to add a Sun light source instead? Maybe add an example with it or just a line or two about how that would look in the asset.

Author: Did not want to add an example until it's working well. Also, ideally I would want to have another page on light sources in general to refer to instead

<details markdown=block><summary markdown=span>More info</summary>

To give advanced users some control of that performance, tessellation can be enabled/disabled for each GeoJson layer. The degree of tessellation can also be controlled. See properties under the "Tessellation" owner for more details.

The tessellation properties can also be set per feature in the .geojson file. Here are the details for the available properties:

| Property Identifier              | Accepted keys (in .geojson file) | Value                    | Description |
| -------------------              |--------------------------------- | ------------------------ | ----------- |
| Tessellation.Enabled              | `tessellate`                     | boolean (`true`/`false`)| Decide whether or not to use tessellation. Default is `true` |
| Tessellation.TessellationDistance | `tessellationDistance`           | float, (> 0)            | The distance/resolution for the tessellation. Anythin larger than this distance will be split into smaller pieces |

> | Tessellation.TessellationLevel    | `tessellationLevel`              | int                     | Decides how much to tessellate, in combination with the tessellation distance. The resulting distance used for the "splitting" is the tessellation distance divided by this value, so the larger the value, the smaller the pieces (and the more the performance will eb affected)  |

Reviewer: ```suggestion
| Tessellation.TessellationLevel    | `tessellationLevel`              | int                     | Decides how much to tessellate, in combination with the tessellation distance. The resulting distance used for the "splitting" is the tessellation distance divided by this value, so the larger the value, the smaller the pieces (and the more the performance will be affected)  |
```

*lines.geojson*
```json
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {
        "stroke": "#e849fd",
        "stroke-width": 2,
        "stroke-opacity": 1
      },
      "geometry": {
        "type": "LineString",
        "coordinates": [[ -68.159, -16.426 ], [ -65.170, -19.062 ], [ -64.775, -21.535 ], [ -68.994, -22.391 ], [ -65.083, -15.496 ], [ -62.622, -15.665 ] ]
      }
    }
  ]
}
```

And an asset that loads the files, adds it to Earth, as well as specifies some default properties, to use for the properties that were not specified in the .geojson file

*lines.asset*
```lua
local ExampleLines = {
  Identifier = "Lines-Example",
  File = "lines.geojson",
  HeightOffset = 2000.0, -- Put the geomoetry 2000 meters higher than what's specified in the file
  DefaultProperties = {
    LineWidth = 2.0,
    Extrude = true
  },
  Name = "Example Lines"
}

asset.onInitialize(function()
  openspace.globebrowsing.addGeoJson("Earth", ExampleLines)
end)

asset.onDeinitialize(function()
  openspace.globebrowsing.deleteGeoJson("Earth", ExampleLines)
end)
```

> More examples are available in the OpenSpace/data/assets/examples folder of your OpenSpace installation.

Reviewer: ```suggestion
More examples are available in the OpenSpace/data/assets/examples/geojson folder of your OpenSpace installation.
```

Some examples of features are shown below (polygons, points with textures, and lines).

![geojson examples](./images/geojson/geojson_examples.jpg)

Each GeoJson file loads a *collection* of features, which will make up a group of rendered features in OpenSpace, that can be toggled on/off, et cetera. In OpenSpace, the contents of a file are loaded into one property owner that will show up under the globe in the user interface, under "GeoJson". Due to the similarity to [Globebrowsing Layers](./working-with-layers.md), we refer to these as "GeoJson Layers", or "Globe Geometry Layers".

## Adding GeoJson features to a globe
There are two ways of adding GeoJson features to a selected globe: either directly from a .geojson file or by specifying a table in an OpenSpace asset. Once added, all content in a GeoJson file will appear as a component under "GeoJson" in the scene menu, as part of the globe it was added to.

### Creating a GeoJson asset

> 

Reviewer 2: ```suggestion
```

Reviewer 2: Just for compactness and consistency with the previous headings

The preferred way to load a GeoJson file is to create an asset. This lets you load the file and add it to the correct globe, already at startup. It also lets you easily customize the visual properties of the rendered feature.

Adding GeoJson to a globe through an asset works very similarly to how a Globebrowsing Layer is added. An example is shown in the [Example](#example) section below.

### Add directly from a file, using drag and drop

> 

Reviewer 2: ```suggestion
```
